src/sentry/event_manager.py

```diff
@@ -76,6 +84,146 @@ def get_fingerprint_for_event(event):
     return fingerprint
 
 
+def is_invalid_stack(frames):
+    if len(frames) != 1:
+        return False
+
+    frame = frames[0]
+    abs_path = frame.get('abs_path', '')
+    return not frame.get('function') and (abs_path.startswith('blob:') or is_url(abs_path))
+
+
+def get_frame_hash(platform, frame):
```

Comment:
i might try to use this on the frame interface to avoid duplication depending on how much it winds up differing from what we're doing on processed frames